lib/constants.go

```diff
@@ -1,12 +1,20 @@
 package libunlynx
 
 import (
+	"os"
 	"sync"
 	"time"
 
 	"go.dedis.ch/onet/v3/simul/monitor"
 )
 
+func init() {
+	tmp, err := time.ParseDuration(os.Getenv("MEDCO_TIMEOUT"))
+	if err != nil {
+		TIMEOUT = tmp
```

Comment:
Last comment:
```
if len(os.Getenv("MEDCO_TIMEOUT") > 0 {
  log.Warn("Couldn't parse MEDCO_TIMEOUT")
}
```

Reply:
I've added it in 5564657
However now that I updated to the new onet version 3.1.1 the error with the `"E : overlay.go:613 (v3.(*Overlay).nodeDelete) - Error while closing node: Can't shutdown empty ProtocolInstance:"` pops way more often. Do you know what changed with this new 3.1 version of Onet?

<img width="959" alt="Screenshot 2020-03-11 at 14 42 22" src="https://user-images.githubusercontent.com/9702328/76423074-88427880-63a6-11ea-8505-5cfdd4d41f2c.png">

Reply:
It disappears if I remove the error condition when calling `tn.SetConfig()`

Reply:
Do you have a simple test-case that I can run to trigger it?

And what do you mean by "removing the error condition when calling `tn.SetConfig()`"?

Reply:
@ineiti  see here https://github.com/ldsec/unlynx/pull/38#discussion_r391641523

Reply:
fixed https://github.com/ldsec/unlynx/pull/38#discussion_r391672676